pkg/controller/networkpolicy/validate.go

```diff
@@ -118,11 +118,16 @@ func (v *NetworkPolicyValidator) Validate(ar *admv1.AdmissionReview) *admv1.Admi
 }
 
 // validateAntreaPolicy validates the admission of a Antrea NetworkPolicy CRDs
-func (v *NetworkPolicyValidator) validateAntreaPolicy(op admv1.Operation, tier string) (string, bool) {
+func (v *NetworkPolicyValidator) validateAntreaPolicy(op admv1.Operation, tier string, ingress, egress []secv1alpha1.Rule) (string, bool) {
 	allowed := true
 	reason := ""
 	switch op {
 	case admv1.Create, admv1.Update:
+		if ruleNameUnique := v.validateRuleName(ingress, egress); !ruleNameUnique {
+			allowed = false
+			reason = fmt.Sprint("Rules names must be unique within the policy")
```

Comment:
i think reason should be lower case

```suggestion
			reason = fmt.Sprint("rules names must be unique within the policy")
```

Reply:
Yeah